START_HERE.ipynb

```diff
@@ -13,15 +13,15 @@
    "id": "0fe3269c-7454-451a-8d88-d775a6e7de93",
    "metadata": {},
    "source": [
-    "[General Startup Guide](opal/starter-notebooks/startup-guide/Startup_Guide_Jupyterlab.ipynb)"
+    "[General Startup Guide](starter-notebooks/startup-guide/Startup_Guide_Jupyterlab.ipynb)\n"
    ]
   },
   {
    "cell_type": "markdown",
```

Comment:
Replace the "Starter Notebooks" section with something explaining that weave is used to upload and access data, see weave README. And Provide the weave README link.